src/backup/after_backup_action_handler.py

```diff
@@ -52,20 +52,21 @@ def __process(self, request_body_json):
             ErrorReporting().report(error_message)
             return
 
-        backup_table_metadata = BigQueryTableMetadata.get_table_by_reference(
-            copy_job_results.target_table_reference)
+        backup_table_metadata = BigQueryTableMetadata.get_table_by_big_query_table(
```

Comment:
You know it will return BigQueryTableMetadata, so maybe just 'from_big_query_table'?

Reply:
Or just 'from_bq_table'

Reply:
Im leaving it as it is, because this method name follows convention in BigQueryTableMetadata and I don't want to refactor the whole class.